package.json

```diff
@@ -46,17 +46,15 @@
     "node": ">=8"
   },
   "files": [
-    "error.html",
-    "plugins/BinaryMediaTypes.js",
-    "StaticFileHandler.js"
+    "src/error.html",
+    "src/plugins/BinaryMediaTypes.js",
+    "src/StaticFileHandler.js"
   ],
   "scripts": {
-    "test": "nyc --cwd ./ ./node_modules/.bin/mocha",
-    "lint": "prettier -l \"../**/*.{js,md,yml,json,html}\"",
-    "lint-fix": "prettier --write \"../**/*.{js,md,yml,yaml,json,html}\"",
-    "coverage": "npm run test && nyc report --reporter=text-lcov | coveralls",
-    "prepack": "cp ../README.md ./ ; cp ../LICENSE ./",
```

Comment:
I don't think this is needed anymore, so I removed it :wave: 
When running `npm pack`  the README.md and LICENSE are in the `tgz` file. 

Leaving this in just duplicated the README & LICENSE files, that were twice in the `tgz`, both at root dir and src/

Reply:
That's correct. It is not needed. This was a wonky hack to support the files being in `./src/`